HLTrigger/JetMET/plugins/HLTJetHFCleaner.h

```diff
@@ -0,0 +1,216 @@
+      auto passSigmaEtaPhiCut = (sigmaEtaEta - sigmaPhiPhi) < sigmaEtaPhiDiffThresh_;
+
+      if (applySigmaEtaPhiCornerCut_) {
+        auto const inCorner = (sigmaEtaEta < cornerCutSigmaEtaEta_) and (sigmaPhiPhi < cornerCutSigmaPhiPhi_);
+        passSigmaEtaPhiCut &= !inCorner;
+      }
+      if (!passSigmaEtaPhiCut) {
+        continue;
+      }
+    }
+
+    // Cut related to central strip size
+    if (applyStripSizeCut_) {
+      auto const& centralEtaStripSizes = iEvent.get(centralEtaStripSizeToken_);
+      if ((centralEtaStripSizes[jetRef] >= centralEtaStripSizeThresh_)) {
+        continue;
+      }
+    }
+
+    cleaned_jets->emplace_back(jet);
+  }
+
+  iEvent.put(std::move(cleaned_jets));
+}
+
+// ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
+template <typename JetType>
+void HLTJetHFCleaner<JetType>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("jets", edm::InputTag("hltAK4PFJetsTightIDCorrected"))->setComment("Input jet collection.");
+  desc.add<edm::InputTag>("mets", edm::InputTag("hltMet"))->setComment("Input MET collection.");
+  desc.add<edm::InputTag>("sigmaEtaEta", edm::InputTag("hltHFJetShowerShape", "sigmaEtaEta"))
+      ->setComment("Input collection which stores the sigmaEtaEta values per jet.");
+  desc.add<edm::InputTag>("sigmaPhiPhi", edm::InputTag("hltHFJetShowerShape", "sigmaPhiPhi"))
+      ->setComment("Input collection which stores the sigmaPhiPhis values per jet.");
+  desc.add<edm::InputTag>("centralEtaStripSize", edm::InputTag("hltHFJetShowerShape", "centralEtaStripSize"))
+      ->setComment("Input collection which stores the central strip size values per jet.");
+  desc.add<double>("jetPtThresh", 100)
+      ->setComment("The minimum pt value for a jet such that the cuts will be applied.");
+  desc.add<double>("dphiJetMetThresh", 2.5)
+      ->setComment(
+          "The minimum value for deltaPhi between jet and MET, such that the cuts will be applied to a given jet.");
+  desc.add<double>("jetEtaMin", 2.9)->setComment("Minimum value of jet |eta| for which the cuts will be applied.");
+  desc.add<double>("jetEtaMax", 5.0)->setComment("Maximum value of jet |eta| for which the cuts will be applied.");
+  desc.add<double>("sigmaEtaPhiDiffThresh", 0.05)
+      ->setComment("Determines the threshold in the following cut: sigmaEtaEta-sigmaPhiPhi < X");
+  desc.add<double>("cornerCutSigmaEtaEta", 0.02)
+      ->setComment(
+          "Corner cut value for sigmaEtaEta. Jets will be cut if both sigmaEtaEta and sigmaPhiPhi are lower than the "
+          "corner value.");
+  desc.add<double>("cornerCutSigmaPhiPhi", 0.02)
+      ->setComment(
+          "Corner cut value for sigmaPhiPhi. Jets will be cut if both sigmaEtaEta and sigmaPhiPhi are lower than the "
+          "corner value.");
+  desc.add<int>("centralEtaStripSizeThresh", 3)
```

Comment:
Please change `Thresh` to `Min` or `Max` depending on the cut, to make more obvious in which direction the cut goes..

Reply:
Thanks, I updated the parameter names as such. Just to note, when I used `centralEtaStripSizeMax` as a variable, I set the default to `2` instead, and allowed jets to pass `centralEtaStripSize <= centralEtaStripSizeMax`, since I thought `Max` value should be allowed. I did the same implementation for `sigmaEtaPhiDiffMax`. Let me know if you have any comments about this, thanks!